app/main.py

```diff
@@ -1,3 +1,27 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from .customer.customer import Customer
+from .car.car import Car
+from .shop.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        data = json.load(file)
+
+    fuel_price = data["FUEL_PRICE"]
+
+    customers = [Customer(name=customer["name"],
+                          products=customer["product_cart"],
+                          location=customer["location"],
+                          money=customer["money"],
+                          car=Car(**customer["car"]))
+                 for customer in data["customers"]]
```

Comment:
Use unpacking (using `**`)
Hint: You can place Car creation inside Customer init